server/plugins/LFUGens.cpp

```diff
@@ -2817,7 +2817,7 @@ static inline bool check_gate_ar(EnvGen * unit, int i, float & prevGate, float *
 
 static inline bool EnvGen_nextSegment(EnvGen * unit, int & counter, double & level)
 {
-	//if (unit->m_stage == ENVGEN_NOT_STARTED) { return true; } // this fixes doneAction 14, but breaks with EnvGen_next_aa
+	if (unit->m_stage == ENVGEN_NOT_STARTED) { counter = INT_MAX; return true; } 
```

Comment:
please break out this block so that each statement (the `if`, the assignment to counter, and the return statement) is on a separate line

Reply:
done

Reply:
I don't see where it is, @sonoro1234 - the code seems unchanged. Maybe you need to push it?